sources/host-ctr/go.mod

```diff
@@ -1,20 +1,20 @@
 module host-ctr
 
-go 1.21.0
+go 1.22.0
 
-toolchain go1.21.4
+toolchain go1.22.2
 
 require (
-	github.com/aws/aws-sdk-go v1.51.2
-	github.com/awslabs/amazon-ecr-containerd-resolver v0.0.0-20240318153621-07ad6deb57bb
-	github.com/containerd/containerd v1.6.30
+	github.com/aws/aws-sdk-go v1.53.10
+	github.com/awslabs/amazon-ecr-containerd-resolver v0.0.0-20240521172427-b580afd02343
+	github.com/containerd/containerd v1.6.32
```

Comment:
In the past, we have not updated this library dependency without updating the package at https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/containerd/Cargo.toml to avoid any possible breaking changes between these two (even if that would be unlikely).

Reply:
Ack, will update the containerd `Cargo.toml` with the `url` and `sha512` of the latest version

Reply:
From our offline discussion, I'll be doing the Go updates in `host-ctr` keeping `containerd` locked at v1.6.30.

`containerd` will be upgraded as part of a separate issue: https://github.com/bottlerocket-os/bottlerocket/issues/3876